app/views/works/_work_activity.html.erb

```diff
@@ -0,0 +1,36 @@
+<div id="activity-history">
+  <% if @work.work_activity.count > 0 %>
+    <h2>Activity History</h2>
+    <div class="form-check form-switch">
+      <input id="show-change-history" class="form-check-input" type="checkbox" role="switch" id="flexSwitchCheckDefault" checked>
+      <label class="form-check-label" for="flexSwitchCheckDefault">Show change history</label>
+    </div>
+    <% @work.activities.each do |activity| %>
+      <% if activity.activity_type == 'CHANGES' %>
+        <div class="activity-history-item activity-history-log-item">
+          <div class="activity-history-log-title">
+            <b><%= activity.created_by_user&.display_name_safe %></b>
+            <span title="<%= activity.created_at.localtime %>"><%= distance_of_time_in_words_to_now(activity.created_at) %></span>
+          </div>
+          <span class="comment-html"><%= activity.message_html.gsub('{USER-PATH-PLACEHOLDER}', users_path).html_safe %></span>
+        </div>
+      <% elsif activity.activity_type == 'FILE-CHANGES' %>
+        <div class="activity-history-item activity-history-log-item">
```

Comment:
I would love to see this as a partial and the parts that are changing based on the type passed in as locals
`render partail: 'activity_log', locals: { user_name: activity.created_by_user&.display_name_safe, time_title: activity.created_at.localtime, time: distance_of_time_in_words_to_now(activity.created_at), comment: activity.message_html.gsub('{USER-PATH-PLACEHOLDER}', users_path).html_safe }`

``` In the partial
<div class="activity-history-item activity-history-comment-item">
      <div class="activity-history-comment-title">
        <b><%= user_name %></b>
        <span title="<%= time_title %>"><%= time %></span>
      </div>
      <span class="comment-html"><%= comment %></span>
    </div>
```

Reply:
I created a separate issue for this: https://github.com/pulibrary/pdc_describe/issues/564